docs/make.jl

```diff
@@ -52,14 +53,37 @@ for (i, f) in enumerate(pluto_files)
     global TutorialMenu
     @info "Building Pluto Notebook $f.jl"
     pluto_file = pluto_src_folder * f * ".jl"
-    s = Pluto.ServerSession()
-    nb = Pluto.SessionActions.open(s, pluto_file; run_async=false)
-    write(pluto_output_folder * f * "_pluto.html", Pluto.generate_html(nb))
+    html = notebook2html(pluto_file)
     write(
         pluto_output_folder * f * ".md",
         """
+        ```@meta
+        EditURL = "$(pluto_src_folder)$(f).jl"
+        ```
         ```@raw html
-        <iframe style="border:none; width:100%; height: $(pluto_heights[i])rem;" src="$(f)_pluto.html"></iframe>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/fonsp/Pluto.jl@0.16.4/frontend/treeview.css" type="text/css" />
+        <style>
+        div.markdown {
+            padding-top: 1rem;
+            padding-bottom: 2rem;
+        }
+        </style>
+        $html
+        <script>
+        require(['jquery', 'katex', 'katex-auto-render'], function(\$, katex, renderMathInElement) {
+            \$(document).ready(function() {
+                renderMathInElement(document.body);
+                /* Apparently, Pluto wraps LaTeX inside a span class 'tex'. */
+                var equations = \$('p.tex');
+                for (var i = 0; i < equations.length; i++) {
+                    var text = equations[i].textContent;
+                    text = text.split('\$').join('');
+                    console.log(text);
+                    render(text, equations[i]);
+                }
+            })
+        })
```

Comment:
I've now changed this to a more elegant KaTeX solution as can be found at <https://rikhuijzer.github.io/PlutoStaticHTML.jl/dev/#LaTeX-equations>. Renders inline and display math both fine: https://huijzer.xyz/posts/correlations/.

Reply:
Ah but KaTeX is already loaded by documenter in the beginning anyways I think? Well, The options part looks much nicer, let's see how that works on CI :) thanks for providing this directly as a suggestion here, I did not see the LaTeX Equations sections since it is under Franklin. One could maybe restructure that to also have that for Documenter-stuff.

Reply:
Oh yes that's very likely. My bad.